lib/FFI/Platypus/Type/Enum.pm

```diff
@@ -248,6 +249,19 @@ Perl:
  
  $ffi->attach( f => [ 'foo_t' ] => 'void' );
 
+=item casing
+
+ $ffi->load_custom_type('::Enum', $name, { casing => 'upper' }, ... ); # version 0.05
+ $ffi->load_custom_type('::Enum', $name, { casing => 'keep'  }, ... ); # version 0.05
+
+[version 0.05]
```

Comment:
```suggestion

[version 0.06]

 $ffi->load_custom_type('::Enum', $name, { casing => 'upper' }, ... );
 $ffi->load_custom_type('::Enum', $name, { casing => 'keep'  }, ... );

```

This is the convention that I use when the whole option is new as version x.yz.  Also this will go into 0.06, so we should mark it as thus.

Reply:
Nice. I was wondering how this was supposed to look. I'll update